src/ResourceManager/TrafficManager/Commands.TrafficManager2/Profile/NewAzureTrafficManagerProfile.cs

```diff
@@ -52,19 +54,37 @@ public class NewAzureTrafficManagerProfile : TrafficManagerBaseCmdlet
         [ValidateNotNullOrEmpty]
         public string TrafficRoutingMethod { get; set; }
 
+        [Alias("ProtocolForMonitor")]
         [Parameter(Mandatory = true, HelpMessage = "The protocol of the monitor.")]
-        [ValidateSet(Constants.HTTP, Constants.HTTPS, IgnoreCase = false)]
+        [ValidateSet(Constants.HTTP, Constants.HTTPS, Constants.TCP, IgnoreCase = false)]
         [ValidateNotNullOrEmpty]
         public string MonitorProtocol { get; set; }
 
+        [Alias("PortForMonitor")]
```

Comment:
Aliases only partially work for tab completion (you can't tab through them by default, but you can give the first letter and have the alias come through).  I don't think we settled on this as a solution, why can't this just be Path, Port, Protocol, since there is no scenario for adding any of these things outside monitoring them?